spec/features/admin/merchants/index_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe "Admin Merchants Index" do
-  let!(:merchant1) {FactoryBot.create(:merchant)}
-  let!(:merchant2) {FactoryBot.create(:merchant)}
-  let!(:merchant3) {FactoryBot.create(:merchant)}
+  let!(:merchant1) {FactoryBot.create(:merchant, name: "Jurassic Quark")}
```

Comment:
Always good to have test instances named differently with different attributes.  The combo of factory bot and choosing specific attributes is great